var/spack/repos/builtin/packages/py-torchvision/package.py

```diff
@@ -118,6 +121,7 @@ class PyTorchvision(PythonPackage):
     depends_on("cuda", when="+nvjpeg")
     depends_on("ffmpeg@3.1:", when="+ffmpeg")
     depends_on("cuda", when="+video_codec")
+    depends_on("giflib", when="+gif")
```

Comment:
TODO: I'm not even sure if it's possible to build against external giflib installations.